images/origin/scripts/recycler.sh

```diff
@@ -22,8 +21,20 @@ if [[ ! -d ${dir} ]]; then
     exit 1
 fi
 
+# remount to nfsv3 so not restricted by nfs v4 acls, see nfs4_acl(5)
+tmp=${dir}
+mnt=`grep -w "${dir}" /proc/mounts |grep -w nfs4`
+exp=""
+[[ ! -z ${mnt} ]] &&  exp=`echo ${mnt}|awk '{print $1}'`
+[[ ! -z ${exp} ]] && tmp=`mktemp -d` && mount ${exp} ${tmp} -o vers=3,nolock
 # shred all files
-find ${dir} -type f -exec shred -fuvz {} \;
+#find ${tmp} -type f -exec shred -fuvz {} \;
```

Comment:
switching from shred to a simple rm can leave data intact on the disk, right? that seems like a problem